presto-main/src/main/java/io/prestosql/security/AccessControlManager.java

```diff
@@ -492,6 +492,23 @@ public void checkCanCreateView(SecurityContext context, QualifiedObjectName view
         }
     }
 
+    @Override
+    public void checkCanRenameView(SecurityContext context, QualifiedObjectName viewName, QualifiedObjectName newViewName)
+    {
+        requireNonNull(context, "context is null");
+        requireNonNull(viewName, "tableName is null");
+        requireNonNull(newViewName, "newTableName is null");
+
+        authenticationCheck(() -> checkCanAccessCatalog(context.getIdentity(), viewName.getCatalogName()));
+
+        authorizationCheck(() -> systemAccessControl.get().checkCanRenameTable(context.toSystemSecurityContext(), viewName.asCatalogSchemaTableName(), newViewName.asCatalogSchemaTableName()));
```

Comment:
Should this be `checkCanRenameView`? Do we have tests for this kind of thing somewhere?